### DISCUSSION TITLE
The IsReadOnly property of DataGridTextColumn cannot use binding, right?

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#14957] [Q&A]
Code like:

`<DataGridTextColumn Header="Title" Binding="{Binding TitleString, Mode=TwoWay}" IsReadOnly="{Binding Selected}"/> ` 
 
will throw an error 'Unable to find suitable setter or adder for property IsReadOnly of type Avalonia.Controls.DataGrid'.

**Comment by IronEaterBeast** [comment]
@timunie hello, can you help me.

**Reply by timunie** [reply]
You can / need to write an Attachedproperty or a Behavior.

**Comment by maxkatz6** [comment]
It's not a styled/direct property, so no - no bindings https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls.DataGrid/DataGridColumn.cs#L502-L524
